platform/integrations/karpenter.mdx

```diff
@@ -0,0 +1,449 @@
+---
+title: Using Karpenter with vCluster Platform
+sidebar_label: Karpenter
+sidebar_position: 7
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Flow, { Step } from "@site/src/components/Flow";
+import Label from "@site/src/components/Label";
+
+vCluster is compatible with Karpenter and vCluster's features can be used in conjuntion with Karpenter to
+further reduce infrastructure costs.
+
+This article will explain how to [Setup Designated Node Pool for a vCluster](#setup-designated-node-pool-for-a-vcluster), [Setup Designated Node
+Pool for a Project](#setup-designated-node-pool-for-a-project), and [Setup Designated Node Pool for Dev/Prod Workloads Accross All vClusters](#setup-designated-node-pool-for-a-project). All
+three of which can be powerful cost saving tools.
+
+This article assumes you are using a host cluster provisioned on one of the following managed Kubernetes
+services: EKS (Elastic Kubernetes Service) or AKS (Azure Kubernetes Service).
+
+Much of this article borrows and builds upon the [Getting Started with Karpenter](https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/) documentation and the [Karpenter Provider Azure README.md](https://github.com/Azure/karpenter-provider-azure).
+
+### Pre-requisites for Setups
+1. A kubernetes cluster setup in either AKS or EKS.
+2. Necessary permissions in chosen cloud provider to use Karpenter.
+3. vCluster Platform installed on host cluster. Read [Deploy the vCluster Platform docs](https://www.vcluster.com/docs/platform/install/quick-start-guide#2-deploy-the-vcluster-platform).
+4. (EKS Only) EBS driver installed. [Get the Amazon EBS CSI driver add-on](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html#adding-ebs-csi-eks-add-on).
+5. (EKS Only) Set the environment variables: `CLUSTER_NAME`, `ARM_AMI_ID`, `AMD_AMI_ID`, `K8S_VERSION`.
+6. (EKS Only) Delete the gp2 StorageClass with `kubectl delete storageclass gp2` and replace with gp3 by applying following yaml:
+```yaml
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: gp3
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: ebs.csi.aws.com
+volumeBindingMode: WaitForFirstConsumer
+parameters:
+  type: gp3
+allowVolumeExpansion: true
+
+```
+
+## Setup Designated Node Pool for a vCluster
+This section utilizes taints and tolerations to guide vcluster pods to the desired node. This works best if every node pool or node utilizes a taint. Otherwise, the pods may be scheduled on non-tainted nodes.
+1. Select "Create Virtual Cluster"
+2. Select "Configure" and add a toleration to virtual cluster pods using the yaml editor. If the below yaml is out of date then see the [applying tolerations docs](https://www.vcluster.com/docs/v0.19/architecture/scheduling#automatically-applying-tolerations-to-all-pods-synced-by-vcluster).
```

Comment:
This references old docs (v0.19) which I wouldn't do in this version of the platform. 

Is there no similar docs in v0.20?

Reply:
No, I could not find anything close and the linked docs are out of date as well, the format for the parameters value has changed.

Reply:
Let's discuss offline what these docs do.